concepticondata/conceptlists/Morucci-2019-643.tsv

```diff
@@ -0,0 +1,644 @@
+Morucci-2019-643-481	481	rotondo	round	0.59	0.308	4.487	0.641	4.923	4.923	vista	10.949	1.149	0.422	6.684	2.882	6.316	1395	ROUND
+Morucci-2019-643-482	482	rotto	broken	0.108	0.216	4.108	2.324	4.784	4.784	vista	11.541	1.174	0.405	6.611	2	6.632	588	BROKEN
+Morucci-2019-643-483	483	rovente	red-hot	1.487	0.872	4.718	0.59	3.026	4.718	tatto	10.692	1.357	0.386	4.889	4.6	5.526		
+Morucci-2019-643-484	484	rubino	ruby	0.051	0.103	0.436	0.128	4.974	4.974	vista	5.692	0.515	0.865	3.444	4.85	4.526		
+Morucci-2019-643-485	485	rumoroso	noisy	0	0.051	0.564	4.974	1.974	4.974	udito	7.564	0.854	0.658	6	2.824	6.316		
+Morucci-2019-643-486	486	ruvido	rough	0.821	0.179	4.974	0.872	3.615	4.974	tatto	10.462	1.197	0.458	5.889	3.5	6.316	1923	ROUGH
+Morucci-2019-643-487	487	sabbioso	sandy	1.412	0.412	4.735	1.029	4.647	4.735	tatto	12.235	1.307	0.353	5.263	3.588	5.278		
+Morucci-2019-643-488	488	saggio	essay	0.03	0.121	0.156	3.636	2.364	3.636	udito	6.308	0.879	0.572	6.421	4	5.526		
+Morucci-2019-643-489	489	salato	salty	4.949	1.256	0.436	0.053	1.026	4.949	gusto	7.719	1.045	0.634	6.941	2.778	6.833	1091	SALTY
+Morucci-2019-643-490	490	saldo	balance	0.026	0.103	4.256	0.897	3.744	4.256	tatto	9.026	1.017	0.469	5.526	4.824	4.944		
+Morucci-2019-643-491	491	salmastro	brackish	3.289	2.865	1.289	0.351	2.632	3.289	gusto	10.427	1.439	0.282	3.526	5.529	4.158	70	BRACKISH
+Morucci-2019-643-492	492	sanguinante	bleeding	1.892	2.054	2.27	0.514	4.972	4.972	vista	11.702	1.419	0.381	5.722	3.8	5.947		
+Morucci-2019-643-493	493	saponoso	soapy	1.846	2.923	3.333	0.179	3.59	3.59	vista	11.872	1.416	0.287	2.556	4.55	3.421		
+Morucci-2019-643-494	494	saporito	tasty	4.974	3.59	0.641	0.256	2.692	4.974	gusto	12.154	1.296	0.388	6.211	3.588	6.421		
+Morucci-2019-643-495	495	sassoso	stony	0.184	0.132	4.211	1.684	4.737	4.737	vista	10.947	1.14	0.421	3.389	4.211	3.842		
+Morucci-2019-643-496	496	saturo	saturated	1.656	1.562	1.625	1.156	3.719	3.719	vista	9.719	1.515	0.264	4.667	6.25	4.263		
+Morucci-2019-643-497	497	sbagliato	wrong	1.818	1.606	2.061	2.906	3.212	3.212	vista	11.603	1.573	0.138	6.882	2.611	6.056	1390	WRONG
+Morucci-2019-643-498	498	sbiadito	faded	0.132	0.103	0.237	0.263	4.667	4.667	vista	5.401	0.576	0.845	6.529	4.611	5.778		
+Morucci-2019-643-499	499	sbilenco	lopsided	0.031	0.031	2.625	0.375	4.75	4.75	vista	7.812	0.859	0.604	4	5.1	3.737		
+Morucci-2019-643-500	500	sbriciolabile	crumble	1.769	0.462	4.667	1.615	4.077	4.667	tatto	12.59	1.393	0.334	2.842	4.125	4.158		
+Morucci-2019-643-501	501	sbrilluccicante	shimmering	0.054	0.108	0.514	0.216	4.811	4.811	vista	5.703	0.604	0.834	4	4.353	4.5		
+Morucci-2019-643-502	502	scampanellante	jingle bell	0	0	0.694	4.944	2.778	4.944	udito	8.417	0.884	0.587	2.333	4.6	2.789		
+Morucci-2019-643-503	503	scarlatto	scarlet	0.103	0.077	0.103	0.128	4.949	4.949	vista	5.359	0.375	0.909	4.706	5.059	4.167		
+Morucci-2019-643-504	504	scarno	bare	0.513	0.205	3.436	0.487	4.333	4.333	vista	8.974	1.127	0.46	4.105	5.765	4.105	1125	BALD
```

Comment:
Not "BALD"; this means some like "bare, meagre, lean", usually as a metaphor for something (like "un articolo scarno" - "a meagre paper")